client/src/components/Sidebar/Component.js

```diff
@@ -11,17 +11,23 @@ const Wrapper = styled.aside`
   border: 1px solid ${props => props.theme.border};
   border-radius: 2px;
   background-color: ${props => props.theme.foreground};
-  
+
   @media (max-width: 768px) {
     display: none;
   }
 `;
 
-const Sidebar = ({ token }) => (
-  <Wrapper>
-    {token && <SidebarCreatePostButton />}
-    <SidebarCategoryList />
-  </Wrapper>
-);
+const Sidebar = props => {
+  useEffect(() => {
+    props.fetchCategories();
+  }, []);
```

Comment:
I'll need to revisit here. I think we will fetch the users *subscribed* categories here, right? Listing all categories won't make sense anyway (maybe make them accessible via search later).

Reply:
Yes, I still haven't decided how I want that to go exactly. Perhaps we add 'front page' and 'popular' pages like Reddit for discovering new ones. Also I was thinking we should rename categories into communities maybe.

Reply:
I'd go for front page and popular. "all" already sounds strange. updated it in this pull request already. Communities sounds good!